plutus-playground/plutus-playground-lib/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright Author name here (c) 2018
```

Comment:
I feel you're giving "Author name here" too much credit.

Reply:
fixed